src/components/filters/filter_icons_overlay/filter_icons_overlay.ts

```diff
@@ -1,28 +1,29 @@
 import { Component } from "@odoo/owl";
-import { CellPosition, DOMCoordinates, SpreadsheetChildEnv } from "../../../types";
+import { CellPosition, SpreadsheetChildEnv } from "../../../types";
 import { GridCellIcon } from "../../grid_cell_icon/grid_cell_icon";
 import { FilterIcon } from "../filter_icon/filter_icon";
 
 interface Props {
-  gridPosition: DOMCoordinates;
+  onMouseDown: (ev: MouseEvent, modifiers: { closePopover: boolean }) => void;
```

Comment:
out of the scope of the task but I wonder if it's a good idea to pass our event to a handler of the parent. It feels like it could lead to weird side effects where the parent would end up trusting the event as if they were to actual target and start handling the ev.offset*/ev.client* which might not be aligned with the parent

Reply:
Yeah that could lead to errors. On the other hand, the parent trusting the event of a child on which it has no control of seems like a programming error. 

We could do something like sanitizing the event and removing the target/offset, whild keeping the "absolute" event modifiers. I don't think it's worth tho.

Reply:
turns out it's not such. a good idea :/ https://www.odoo.com/web#id=3868838&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form

Reply:
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/1915d95b-4e7c-4839-9d34-7cde022f70bb)